src/Model/File.php

```diff
@@ -124,4 +134,24 @@ public function createFromPath(string $path, string $fileName = null): Model
 
         return $entity;
     }
+
+    /**
+     * Useful method to clean up all draft files.
+     * Can be called as user action or on schedule bases to clean up filestore repository.
+     *
+     * @return $this
+     */
+    public function cleanupDrafts(): Model
+    {
+        $m = $this->isEntity() ? $this->getModel() : $this;
```

Comment:
should be allowed to be called on model only ;-)

Reply:
Looks like Crud UserAction callback pass entity not model in there as $this.

Reply:
make sure https://github.com/atk4/data/blob/06dc74732ab2f51a5bfe099e610fefe0c8dce058/src/Model/UserAction.php#L33 is set correctly

Reply:
adressed